terraform/production/main.tf

```diff
@@ -36,5 +38,5 @@ module "cloudfront-production" {
   cost_code= "B0811"
   project_name= "Developer Hub"
   use_cloudfront_cert = false
-  hackney_cert_arn = "arn:aws:acm:us-east-1:153306643385:certificate/71728a39-cd3e-4570-a440-e87f84ef9a0d"
+  hackney_cert_arn = "arn:aws:acm:us-east-1:${data.aws_caller_identity.current.account_id}:certificate/3e7dd3eb-f1b1-480f-aab5-07dc4d9ac5b3"
```

Comment:
Can you please make the certificateId a variable (similar to the way you did for tech radar)

Reply:
The reason it's a variable in the tech radar is that the tech radar is using a modified version of the CloudFront distribution module from common-terraform (because it has the playbook and tech radar on the same domain).

Because the dev hub is just using the module, I don't see any reason to make it a variable?